src/task_queue/tasks.py

```diff
@@ -5,38 +5,48 @@
 from task_queue.helpers import is_time_to_run
 from .jobs import FUNCTIONS
 from api.services.utils import send_slack_message
-from _main_.settings import SLACK_SUPER_ADMINS_WEBHOOK_URL, IS_PROD
+from _main_.settings import SLACK_SUPER_ADMINS_WEBHOOK_URL, IS_PROD, IS_LOCAL, IS_CANARY
 
+ENV = "API Local" if IS_LOCAL else "API CANARY" if IS_CANARY else "API PROD" if IS_PROD else "API DEV"
```

Comment:
unless things have changed, IS_LOCAL means the API is using the local database.  It can be running locally but pointing to a DEV or CANARY database.  Or has that been changed?

Reply:
Hi @BradHN1 I believe I answered this question last week, but for visibility, this new implementation is meant to prevent manually changing environments before deployments but reading them programmatically after they have been passed as env vars to our deployment commands.

We'll be adding the ability to point the local to another ENVs we want next week.